docs/docs-content/architecture/grps-proxy.md

```diff
@@ -11,19 +11,74 @@ sidebar_custom_props:
 
 Palette uses [gRPC](https://grpc.io) to communicate between the management platform and the workload cluster. gRPC is a
 high-performance, open-source universal Remote Procedure Call (RPC) framework. It is used to build distributed
-applications and services. gRPC is based on HTTP/2 and uses protocol buffers ([protobuf](https://protobuf.dev/)) as the
-underlying data serialization framework.
+applications and services. gRPC is based on HTTP/2 protocol and uses protocol buffers
+([protobuf](https://protobuf.dev/)) as the underlying data serialization framework.
 
-:::info
+:::tip
 
 Refer to the [Network Ports](networking-ports.md) documentation for a detailed network architecture diagram with gRPC
 and to learn more about the ports used for communication.
 
 :::
 
+## gRPC and WebSocket
+
+The Palette agent will automatically attempt to connect to the management plane using gRPC through HTTPS using the
+HTTP/2 protocol. In some environments, the network configuration may not allow gRPC traffic to pass through. A common
+scenario is when the network is behind a proxy server that does not support HTTP/2. In this scenario, the Palette agent
+will first attempt to connect to the management plane using HTTP/2. After three failed attempts, the agent will fall
```

Comment:
currently ally retries 10 times. This is subjected to change so can we say 'several' instead of 'three'?

Reply:
Thank you @guyni will change it to reflect the updated value.